readme.txt

```diff
@@ -2,9 +2,9 @@
 Contributors: automattic, mikejolley, jameskoster, claudiosanches, kloon, rodrigosprimo, peterfabian1000, vedjain, jamosova, obliviousharmony
 Tags: e-commerce, store, sales, sell, woo, shop, cart, checkout, downloadable, downloads, payments, paypal, storefront, stripe, woo commerce
 Requires at least: 5.0
-Tested up to: 5.3
+Tested up to: 5.4
 Requires PHP: 7.0
-Stable tag: 4.0.0
+Stable tag: 4.1.0
```

Comment:
Shouldn't this point to 4.0.1?

Reply:
ah right. updated in https://github.com/woocommerce/woocommerce/pull/26097/commits/8e2a6b2c8233fca13b16814080f94408aad9d029.